database/postgres/postgres.js

```diff
@@ -0,0 +1,134 @@
+const { Sequelize, DataTypes } = require('sequelize');
+const { Client } = require('pg');
+
+const db = new Sequelize('test', 'dharmon', null, {
+  host: 'localhost',
+  port: 5432,
+  dialect: 'postgres',
+  logging: false,
```

Comment:
If possible, user variables and read it from config file

Reply:
Great idea, I started with this as a temporary measure intending to change it later, but I'm slowly learning that temporary measures have a way of being left in the code..best to do it right the first time i suppose